src/Notification/Command/ReadNotificationHandler.php

```diff
@@ -10,10 +10,25 @@
 namespace Flarum\Notification\Command;
 
 use Carbon\Carbon;
+use Flarum\Notification\Event\Read;
 use Flarum\Notification\Notification;
+use Illuminate\Contracts\Events\Dispatcher;
 
 class ReadNotificationHandler
 {
+    /**
+     * @var DIspatcher
```

Comment:
```suggestion
     * @var Dispatcher
```

Reply:
See, I was just testing to see if anyone noticed 🙈